python/tidy/servo_tidy/tidy.py

```diff
@@ -474,6 +474,8 @@ def check_rust(file_name, lines):
 
                 if not whitespace:
                     import_block = False
+        if re.search(r".*///.*(TODO|NOTE|FIXME).*", line) or re.search(r".*//!.*(TODO|NOTE|FIXME).*", line):
```

Comment:
This should be merged with the other regex checks [here](https://github.com/servo/servo/blob/556a46f537009bfc0c5cffadcd1a640c2b0c3029/python/tidy/servo_tidy/tidy.py#L492). Also, the pattern could be much simpler like,

``` python
re.search(r'//[/!].*(TODO|FIXME|NOTE)', line)
```